WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -2673,16 +2765,28 @@ public void onScrollToTop() {
     public static void resetLastUpdateDate() {
         mLastAutoUpdateDt = null;
     }
+
+    private boolean isCurrentTagTheFollowingTag() {
```

Comment:
Good question 😊. Trying to give some high level context. 

### Before IA reader changes
All Tags were at the same level selectable by a spinner. 

### After IA reader changes
Now there are two level of filtering:
1. tabs (1st level)
2. subfilter component (2nd level). This only available in the `FOLLOWING` tab

Also we introduced the possibility to filter and navigate posts from a single site/blog directly in the `FOLLOWING` tab. 

Given this, we can have **different kind of tags categories**:
1. **special tags** that are hosted in the tabs 1st level filter (namely `DISCOVER`, `LIKES`, `SAVED`)
2. tags related to **custom lists of tags** (tbh I think these are no more supported (that is new users cannot create them) but it’s possible some users still have them configured). These or other special TAGS like `AUTOMATTIC` tag are hosted to the right of `SAVED` tab as additional tabs themselves.
3. Then we have the `FOLLOWING` tab that hosts
    a. Unfiltered posts from all followed sources (when no subfilter is selected)
    b. Filtered streams of posts as filtered from the bottomsheet subfilter SITES/TAGS lists

So in this context, the `currentTag` behaves as usual when you are in a tab other than `FOLLOWING` or you are in `FOLLOWING` tab without a 2nd level subfilter selected. 

When in `FOLLOWING` tab, the  currentTag can additionally be set to a tag from the list of followed tags or not be valid at all since the reader is set to show posts from a site (in this last case the `mPostListType` is also set to `ReaderPostListType.BLOG_PREVIEW`
 instead of `ReaderPostListType.TAG_FOLLOWED`).

[Note: not directly related to this PR but maybe could be helpful to see how the post list type is set in `ReaderPostListViewModel.onSubfilterChanged`]

Hope the above helps more than confuse; this said, the answer this method should answer is if currentTag is one of those for which we should be in the `FOLLOWING` tab, so we can maybe change the name from `isCurrentTagTheFollowingTag` to `isCurrentTagInFollowingTab`? (if it makes a better sense at all 😊). Changed [here](bb7b190e0e706e0be35ef0dc45f61ac64c5aefbf) 

Note for myself: make a flowchart/scheme of the above at least to document the end of the project 😊.